mitodl/micromasters
# Link channels to users and add all staff as moderators of channel

#### What are the relevant tickets?
Fixes #3574 

#### What's this PR do?
 - Adds many to many relationship between `Channel` to `Program`
 - Users who are staff or instructors of any program linked to the channel are added as moderators when the channel is created
 - When a user's profile is updated and they are moderator of a channel, we do not update their subscription status for that channel

#### How should this be manually tested?
Create a channel and verify that you see the channel in your subscribed channels list afterwards. Update your profile and verify that the channel is still in your subscribed channels.
